crates/subspace-runtime/src/lib.rs

```diff
@@ -988,8 +988,12 @@ impl_runtime_apis! {
     }
 
     impl sp_objects::ObjectsApi<Block> for Runtime {
-        fn extract_block_object_mapping(block: Block, successful_calls: Vec<Hash>) -> BlockObjectMapping {
-            extract_block_object_mapping(block, Some(successful_calls))
+        fn extract_block_object_mapping(block: Block, _successful_calls: Vec<Hash>) -> BlockObjectMapping {
+            // TODO: this is breaking change and cannot be updated without a fork in one step
```

Comment:
I think this should be the way it was before this PR, old clients will not create objects mappings and new clients will, this is fine.